lib/orbit/certification.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Orbit.Certification do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Orbit.RailLine
+
+  # Over in Glides "rail" is referred to as "operator", but it's the same thing.
+  @type certification_type :: :right_of_way | :rail
+
+  @type t :: %__MODULE__{
+          badge: String.t(),
+          type: certification_type(),
+          rail_line: RailLine.t() | nil,
+          expires: Date.t()
+        }
+  schema "certifications" do
+    field(:badge, :string)
+    field(:type, Ecto.Enum, values: [right_of_way: 0, rail: 1])
```

Comment:
Most places we use `Ecto.Enum`, we save the values as atoms (which get converted to strings in the database), though I see the Glides code this is copied from uses ints (it's the only place that uses ints).

Have you thought about whether you'd rather use ints or atoms/strings here?